packages/teleport/src/Databases/DatabaseList/DatabaseList.tsx

```diff
@@ -124,6 +139,24 @@ function LabelCell(props) {
   return renderLabelCell(tags);
 }
 
+function ConnectDBButton(props) {
```

Comment:
I'd just named this `ConnectButton`, its defined in `DatabaseList.tsx`, we can infer it means to connect to a database